src/main/java/com/wuest/prefab/ModRegistry.java

```diff
@@ -88,16 +86,17 @@ public ItemStack makeIcon() {
     public static final RegistryObject<BlockGlassSlab> GlassSlab = BLOCKS.register("block_glass_slab", () -> new BlockGlassSlab(Block.Properties.copy(Blocks.GLASS)));
-    public static final RegistryObject<BlockShaped> PileOfBricks = BLOCKS.register("item_pile_of_bricks", () -> new BlockShaped(BlockShaped.BlockShape.PileOfBricks, AbstractBlock.Properties.of(Material.CLAY, MaterialColor.COLOR_RED).noOcclusion().isViewBlocking(ModRegistry::never)));
-    public static final RegistryObject<BlockShaped> PalletOfBricks = BLOCKS.register("item_pallet_of_bricks", () -> new BlockShaped(BlockShaped.BlockShape.PalletOfBricks, AbstractBlock.Properties.of(Material.CLAY, MaterialColor.COLOR_RED).noOcclusion().isViewBlocking(ModRegistry::never)));
-    public static final RegistryObject<BlockShaped> BundleOfTimber = BLOCKS.register("item_bundle_of_timber", () -> new BlockShaped(BlockShaped.BlockShape.BundleOfTimber, AbstractBlock.Properties.of(Material.WOOD, MaterialColor.COLOR_BROWN).sound(SoundType.WOOD).noOcclusion().isViewBlocking(ModRegistry::never)));
-    public static final RegistryObject<BlockShaped> HeapOfTimber = BLOCKS.register("item_heap_of_timber", () -> new BlockShaped(BlockShaped.BlockShape.HeapOfTimber, AbstractBlock.Properties.of(Material.WOOD, MaterialColor.COLOR_BROWN).sound(SoundType.WOOD).noOcclusion().isViewBlocking(ModRegistry::never)));
-    public static final RegistryObject<BlockShaped> TonOfTimber = BLOCKS.register("item_ton_of_timber", () -> new BlockShaped(BlockShaped.BlockShape.TonOfTimber, AbstractBlock.Properties.of(Material.WOOD, MaterialColor.COLOR_BROWN).sound(SoundType.WOOD).noOcclusion().isViewBlocking(ModRegistry::never)));
-    public static final RegistryObject<Block> EmptyCrate = BLOCKS.register("item_wooden_crate", () -> new Block(AbstractBlock.Properties.of(Material.WOOD).sound(SoundType.WOOD)));
-    public static final RegistryObject<Block> CartonOfEggs = BLOCKS.register("item_carton_of_eggs", () -> new Block(AbstractBlock.Properties.of(Material.WOOD).sound(SoundType.WOOD)));
-    public static final RegistryObject<Block> CrateOfPotatoes = BLOCKS.register("item_crate_of_potatoes", () -> new Block(AbstractBlock.Properties.of(Material.WOOD).sound(SoundType.WOOD)));
-    public static final RegistryObject<Block> CrateOfCarrots = BLOCKS.register("item_crate_of_carrots", () -> new Block(AbstractBlock.Properties.of(Material.WOOD).sound(SoundType.WOOD)));
-    public static final RegistryObject<Block> CrateOfBeets = BLOCKS.register("item_crate_of_beets", () -> new Block(AbstractBlock.Properties.of(Material.WOOD).sound(SoundType.WOOD)));
+
+    public static final RegistryObject<BlockShaped> PileOfBricks = BLOCKS.register("item_pile_of_bricks", () -> new BlockRotatableHorizontalShaped(BlockShaped.BlockShape.PileOfBricks, AbstractBlock.Properties.of(Material.CLAY, MaterialColor.COLOR_RED).noOcclusion().isViewBlocking(ModRegistry::never)));
```

Comment:
The class type within the "RegistryObject" should match the type of class we are creating. This way any future usage of this can use all properties. In this case it should be "public static final RegistryObject<BlockRotatableHorizontalShaped>"